main.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-import "github.com/alexandrst88/terraform-variables-generator/cmd"
+import "github.com/marceloalmeida/terraform-variables-generator/cmd"
```

Comment:
?

Reply:
It was to be able to perform `go get...` on multiple places and build using my changes, just reverted.